src/py21cmmc/mcmc.py

```diff
@@ -183,9 +270,9 @@ def run_mcmc(
             if hasattr(lk, "_simulate") and lk._simulate:
                 logger.warning(
                     """
-Likelihood {} was defined to re-simulate data/noise, but this is incompatible with
-`continue_sampling`. Setting simulate=False and continuing...
-"""
+                    Likelihood {} was defined to re-simulate data/noise, but this is incompatible with
+                    `continue_sampling`. Setting simulate=False and continuing...
+                    """
```

Comment:
This was my bad, but I think this would be better as single " quotes, one pair for each line, i.e.:

```python
logger.warning(
    f"Likelihood {lk} was defined to re-simulate data/noise, but this is incompatible with"
    "`continue_sampling`. Setting simulate=False and continuing..."
)
```